manifests/mod/authn_dbd.pp

```diff
@@ -29,13 +29,13 @@
 #
 class apache::mod::authn_dbd (
   Optional[String] $authn_dbd_params,
-  String $authn_dbd_dbdriver                  = 'mysql',
-  Optional[String] $authn_dbd_query           = undef,
-  Variant[String,Integer] $authn_dbd_min      = '4',
-  Variant[String,Integer]  $authn_dbd_max     = '20',
-  Variant[String,Integer]  $authn_dbd_keep    = '8',
-  Variant[String,Integer]  $authn_dbd_exptime = '300',
-  Optional[String] $authn_dbd_alias           = undef,
+  String $authn_dbd_dbdriver                               = 'mysql',
+  Optional[String] $authn_dbd_query                        = undef,
+  Variant[Integer, Pattern[/^[0-9]+$/]] $authn_dbd_min     = 4,
+  Variant[Integer, Pattern[/^[0-9]+$/]] $authn_dbd_max     = 20,
+  Variant[Integer, Pattern[/^[0-9]+$/]] $authn_dbd_keep    = 8,
+  Variant[Integer, Pattern[/^[0-9]+$/]] $authn_dbd_exptime = 300,
```

Comment:
I'd consider a breaking change and make it `Integer[0]` (assuming 0 really is the lower bound).

Reply:
Neither do I know the correct Integer-Ranges.  Possibly there is a `-1` allowed somewhere?  I don't know.  
But I am with you, removing the `Pattern`.  Give me a „go“.

Reply:
My point was that if it's just numbers then reduce it to just making it `Integer`, possibly with a lower bound. We're doing a breaking change anyway so it's ok to be backwards incompatible.